packages/table/test/selectableTests.tsx

```diff
@@ -31,13 +31,15 @@ describe("DragSelectable", () => {
 
     it("does click selection", () => {
         const onSelection = sinon.spy();
+        const onFocus = sinon.spy();
```

Comment:
Selection can be disabled as well. The determination of whether or not the focus or selection is valid happens in the table parent -- it does the filtering. Not the child. At least that's how the selection logic works, which I borrowed from.